### ISSUE TITLE
Would you be willing to take ownership of google-apis-rs?

### ISSUE THREAD

**Issue by IvanUkhov** [googleapis/google-cloud-rust#16]
Hello,

There is this excellent project that has just been deprecated:

https://github.com/Byron/google-apis-rs/

The background is given in these discussions:

* https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/Can.20I.20reclaim.203.20Google.20CLI.20crates.2Cpublished.20by.20someone.20else.3F
* https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/On.20Google.20APIs.20and.20Sebastian.20Thiel's.20crates

The Rust ecosystem is risking to be left without a comprehensive way to interact with Google APIs. There are other initiatives but they are not this rigorous in terms of the coverage. This page gives a nice overview of what `google-apis-rs` is capable of:

https://byron.github.io/google-apis-rs/

Would you be willing to adopt the repository and maintain it moving forward, with the community's effort of course?

Thank you!

/cc @danoscarmike, @codyoss

**Comment by coryan** [comment]
TL;DR; no, I do not think we can help here.

First, apologies for the late response. We only recently restarted the project to create a Google Cloud SDK for Rust.

We did look at `Byron/google-apis-rs`.  We decided to on in a different direction.  This is **not** a reflection on the quality or features available in said project. We just have different priorities and other considerations. For example, we need tooling that is consistent across more languages than Rust.

I do not want to be ambiguous or unclear: we are aiming to support all the *Google Cloud* APIs. That leaves some APIs (like Ads) that would not be included in this project. For Google Cloud, we hope to make the APIs so good that people would want to move to the crates published out of `google-cloud-rust`.

For APIs beyond Cloud: the tooling we are creating should be able to support those APIs too, and we would support any community efforts to support those additional APIs. We cannot commit to supporting said APIs ourselves.  And please give us some time to get Cloud working before we explore Ads or Geo or something else.
